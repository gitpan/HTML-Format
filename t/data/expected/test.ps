%!PS-Adobe-3.0
%%Creator: HTML::FormatPS (v2.05, using HTML::TreeBuilder v4.1, and HTML::Parser v3.68)
%%CreationDate: Mon Feb 14 20:03:30 2011
%%Pages: 1
%%PageOrder: Ascend
%%Orientation: Portrait
%%DocumentMedia: Plain 595 841 0 white ()
%%DocumentNeededResources: 
%%+ font Courier
%%+ font Times-Bold
%%+ font Times-Roman
%%DocumentSuppliedResources: procset newencode 1.0 0
%%+ encoding ISOLatin1Encoding
%%EndComments

%%BeginProlog
/S/show load def
/M/moveto load def
/SF/setfont load def

%%BeginResource: encoding ISOLatin1Encoding
systemdict /ISOLatin1Encoding known not {
    /ISOLatin1Encoding [
    /space /space /space /space /space /space /space /space
    /space /space /space /space /space /space /space /space
    /space /space /space /space /space /space /space /space
    /space /space /space /space /space /space /space /space

    /space /exclam /quotedbl /numbersign /dollar /percent /ampersand
        /quoteright
    /parenleft /parenright /asterisk /plus /comma /minus /period /slash
    /zero /one /two /three /four /five /six /seven
    /eight /nine /colon /semicolon /less /equal /greater /question
    /at /A /B /C /D /E /F /G
    /H /I /J /K /L /M /N /O
    /P /Q /R /S /T /U /V /W
    /X /Y /Z /bracketleft /backslash /bracketright /asciicircum /underscore
    /quoteleft /a /b /c /d /e /f /g
    /h /i /j /k /l /m /n /o
    /p /q /r /s /t /u /v /w
    /x /y /z /braceleft /bar /braceright /asciitilde /space

    /space /space /space /space /space /space /space /space
    /space /space /space /space /space /space /space /space
    /dotlessi /grave /acute /circumflex /tilde /macron /breve /dotaccent
    /dieresis /space /ring /cedilla /space /hungarumlaut /ogonek /caron

    /space /exclamdown /cent /sterling /currency /yen /brokenbar /section
    /dieresis /copyright /ordfeminine /guillemotleft /logicalnot /hyphen
        /registered /macron
    /degree /plusminus /twosuperior /threesuperior /acute /mu /paragraph
        /periodcentered
    /cedillar /onesuperior /ordmasculine /guillemotright /onequarter
        /onehalf /threequarters /questiondown
    /Agrave /Aacute /Acircumflex /Atilde /Adieresis /Aring /AE /Ccedilla
    /Egrave /Eacute /Ecircumflex /Edieresis /Igrave /Iacute /Icircumflex
        /Idieresis
    /Eth /Ntilde /Ograve /Oacute /Ocircumflex /Otilde /Odieresis /multiply
    /Oslash /Ugrave /Uacute /Ucircumflex /Udieresis /Yacute /Thorn
        /germandbls
    /agrave /aacute /acircumflex /atilde /adieresis /aring /ae /ccedilla
    /egrave /eacute /ecircumflex /edieresis /igrave /iacute /icircumflex
        /idieresis
    /eth /ntilde /ograve /oacute /ocircumflex /otilde /odieresis /divide
    /oslash /ugrave /uacute /ucircumflex /udieresis /yacute /thorn
        /ydieresis
    ] def
} if
%%EndResource
%%BeginResource: procset newencode 1.0 0
/NE { %def
   findfont begin
      currentdict dup length dict begin
     { %forall
        1 index/FID ne {def} {pop pop} ifelse
     } forall
     /FontName exch def
     /Encoding exch def
     currentdict dup
      end
   end
   /FontName get exch definefont pop
} bind def
%%EndResource
%%EndProlog

%%BeginSetup
ISOLatin1Encoding/Courier-ISO/Courier NE
/F5/Courier-ISO findfont 10 scalefont def
ISOLatin1Encoding/Times-Bold-ISO/Times-Bold NE
/F4/Times-Bold-ISO findfont 18 scalefont def
ISOLatin1Encoding/Times-Bold-ISO/Times-Bold NE
/F2/Times-Bold-ISO findfont 24 scalefont def
ISOLatin1Encoding/Times-Roman-ISO/Times-Roman NE
/F3/Times-Roman-ISO findfont 10 scalefont def
ISOLatin1Encoding/Times-Roman-ISO/Times-Roman NE
/F1/Times-Roman-ISO findfont 8 scalefont def
%%EndSetup

%%Page: 1 1
%% Title and pageno
F1 SF
565.3 30.0 M(1)S
5.0 30.0 M()S

5.0 704.5 M
F2 SF
(This is a test)S
5.0 683.5 M
F3 SF
(In order to install and use this package you will need Perl version 5.004 or better. You will also need to have the HTML-Tree)S
5.0 672.5 M
(distribution \(version 0.61 or better\) installed. If you intend to use the PostScript formatter you need the Font-AFM distribution. All of)S
5.0 661.5 M
(these should be available from CPAN. In order to install and use this package you will need Perl version 5.004 or better. You will also)S
5.0 650.5 M
(need to have the HTML-Tree distribution \(version 0.61 or better\) installed. If you intend to use the PostScript formatter you need the)S
5.0 639.5 M
(Font-AFM distribution. All of these should be available from CPAN. In order to install and use this package you will need Perl)S
5.0 628.5 M
(version 5.004 or better. You will also need to have the HTML-Tree distribution \(version 0.61 or better\) installed. If you intend to use)S
5.0 617.5 M
(the PostScript formatter you need the Font-AFM distribution. All of these should be available from CPAN.)S
5.0 596.5 M
(In order to install and use this package you will need Perl version 5.004 or better.)S
27.0 575.5 M
(You will also need to have the HTML-Tree distribution \(version 0.61 or better\) installed.)S
newpath 17.2 578.0 1.2 0 360 arc fill
27.0 554.5 M
(If you intend to use the PostScript formatter you need the Font-AFM distribution.)S
newpath 17.2 557.0 1.2 0 360 arc fill
5.0 533.5 M
(All of these should be available from CPAN.)S
5.0 512.5 M
(In order to install and use this package you will need Perl version 5.004 or better. You will also need to have the HTML-Tree)S
5.0 501.5 M
(distribution \(version 0.61 or better\) installed. If you intend to use the PostScript formatter you need the Font-AFM distribution. All of)S
5.0 490.5 M
(these should be available from CPAN.)S
5.0 444.7 M
F4 SF
(Installation)S
5.0 423.7 M
F3 SF
(Just follow the usual procedure:)S
5.0 402.7 M
F5 SF
(   perl Makefile.PL)S
5.0 391.7 M
(   make)S
5.0 380.7 M
(   make test)S
5.0 369.7 M
(   make install)S
5.0 348.7 M
F3 SF
(In order to install and use this package you will need Perl version 5.004 or better. You will also need to have the HTML-Tree)S
5.0 337.7 M
(distribution \(version 0.61 or better\) installed. If you intend to use the PostScript formatter you need the Font-AFM distribution. All of)S
5.0 326.7 M
(these should be available from CPAN.)S
showpage

%%Trailer
%%EOF
