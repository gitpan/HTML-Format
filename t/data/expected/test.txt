     This is a test
     ==============

     In order to install and use this package you
     will need Perl version 5.004 or better. You
     will also need to have the HTML-Tree
     distribution (version 0.61 or better)
     installed. If you intend to use the PostScript
     formatter you need the Font-AFM distribution.
     All of these should be available from CPAN. In
     order to install and use this package you will
     need Perl version 5.004 or better. You will
     also need to have the HTML-Tree distribution
     (version 0.61 or better) installed. If you
     intend to use the PostScript formatter you
     need the Font-AFM distribution. All of these
     should be available from CPAN. In order to
     install and use this package you will need
     Perl version 5.004 or better. You will also
     need to have the HTML-Tree distribution
     (version 0.61 or better) installed. If you
     intend to use the PostScript formatter you
     need the Font-AFM distribution. All of these
     should be available from CPAN.

     In order to install and use this package you
     will need Perl version 5.004 or better.

       * You will also need to have the HTML-Tree
         distribution (version 0.61 or better)
         installed.

       * If you intend to use the PostScript
         formatter you need the Font-AFM
         distribution.

     All of these should be available from CPAN.

     In order to install and use this package you
     will need Perl version 5.004 or better. You
     will also need to have the HTML-Tree
     distribution (version 0.61 or better)
     installed. If you intend to use the PostScript
     formatter you need the Font-AFM distribution.
     All of these should be available from CPAN.


     Installation
     ------------

     Just follow the usual procedure:

        perl Makefile.PL
        make
        make test
        make install

     In order to install and use this package you
     will need Perl version 5.004 or better. You
     will also need to have the HTML-Tree
     distribution (version 0.61 or better)
     installed. If you intend to use the PostScript
     formatter you need the Font-AFM distribution.
     All of these should be available from CPAN.
